### PR TITLE
add odin module target to json file

### PR DESCRIPTION
**Pull request by netanelgonen**
add Pin configuration to MODULE_UBLOX_ODIN_W2 target

### CODE REVIEW

**Comment by netanelgonen** [comment]
Hi @geky  can you please review?

File: mbed_lib.json [hunk 1]
```diff
@@ -30,6 +30,13 @@
              "SPI_MISO": "PTD7",
              "SPI_CLK":  "PTD5",
              "SPI_CS":   "PTD4"
+        },
+        "MTB_UBLOX_ODIN_W2":  
+        {
```
Comment (geky) [inline mbed_lib.json]: nit: could you make the style match?

```
MTB_UBLOX_ODIN_W2": {
```

**Review by geky — APPROVED** [review]
I don't know much about the platform, but it looks good to me :+1:

(one nit)

**Comment by netanelgonen** [comment]
Hi @geky style error fixed.

**Comment by geky** [comment]
Thanks!

**Pull request merged**